tests/models/test_param.py

```diff
@@ -167,8 +167,8 @@ def test_number_param(self):
         p = Param(42, type="number")
         assert p.resolve() == 42
 
-        p = Param(1.0, type="number")
-        assert p.resolve() == 1.0
+        p = Param(1.2, type="number")
+        assert p.resolve() == 1.2
```

Comment:
I am not sure that I need to update it, but using 1.2 is more significant that 1.0 to prove that `number` type accept float values

Reply:
Good idea, since `1.0 == 1` in Python.

Reply:
:heart: